lib/ts/recipe/jwt/utils.ts

```diff
@@ -20,18 +20,18 @@ import { APIInterface, RecipeInterface, TypeInput, TypeNormalisedInput } from ".
 export function validateAndNormaliseUserInput(
     _: Recipe,
     __: NormalisedAppinfo,
-    config: TypeInput
+    config?: TypeInput
 ): TypeNormalisedInput {
-    let jwtValidity = config.jwtValidity;
+    // let providedOverrides = config === undefined ? {} : config.override;
 
     let override = {
         functions: (originalImplementation: RecipeInterface) => originalImplementation,
         apis: (originalImplementation: APIInterface) => originalImplementation,
-        ...config.override,
+        ...config?.override,
     };
 
     return {
-        jwtValidity,
+        jwtValiditySeconds: config?.jwtValiditySeconds,
```

Comment:
Perhaps you should set the large validity time period here in case it's undefined from the user's side. This way:
- In the normalised config, you can always expect to have a defined validity period
- If using this value in multiple places in the future, you don't have to remember to always check if this is undefined and if ye, set it to the large number.

Reply:
Makes sense, done